### PR TITLE
changed main structure

### PR DESCRIPTION
**Pull request by daniel0814**

### CODE REVIEW

File: ola.c [hunk 1]
```diff
@@ -1,7 +1,8 @@
-#include<stdio.h>
+#include <stdio.h>
 
-int main{
+#define MAX 3
```
Comment (AfonsoCunhaReis1997) [inline ola.c]: the value should be 5

File: ola.c [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
+int main(void){
     printf("olaaaaa");
-    return 0;
+    printf("Max: %d", MAX);
```
Comment (AfonsoCunhaReis1997) [inline ola.c:9]: you should return 0;

File: ola.c [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
 }
```
Comment (pedrocsi) [inline ola.c]: parece me bem xD

**Review by pedrocsi — CHANGES_REQUESTED** [review]

File: ola.c [hunk 4]
```diff
@@ -1,7 +1,8 @@
-#include<stdio.h>
+#include <stdio.h>
```
Comment (AfonsoCunhaReis1997) [inline ola.c:2]: include more stuff

**Review by AfonsoCunhaReis1997 — CHANGES_REQUESTED** [review]

File: ola.c (hunk 3 above)
Comment (daniel0814) [reply ola.c]: Nao vou mudar nada

**Review by AfonsoCunhaReis1997 — APPROVED** [review]
lgtm

**Review by pedrocsi — APPROVED** [review]

**Pull request merged**